public/views/checkout.html

```diff
@@ -0,0 +1,227 @@
+        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
+
+        <script type="text/javascript" src="js/children.js"></script>
+
+        <link href="../css/Site.css" rel="stylesheet" />
+        <link href="../css/checkout.css" rel="stylesheet" />        
+    </head>
+
+    <body>
+        <section class="pre-header">
+            <div class="content">
+                <div class="row vcenter">
+                    <div class="col-sm-1 hidden-xs col-sm-offset-1">
+                        <a class="vcenter" href="https://twitter.com/AMGGuatemala">
+                            <img src="../image/btn-pre-twitter.png" class="img-center" />
+                        </a>
+                    </div>
+                    <div class="col-sm-1 hidden-xs">
+                        <a class="vcenter" href="https://www.facebook.com/amgguatemala">
+                            <img src="../image/btn-pre-facebook.png" class="img-center" />
+                        </a>
+                    </div>
+                    <div class="col-sm-1 hidden-xs">
+                        <a class="vcenter" href="https://www.youtube.com/channel/UCDRWMZkdIWYWq7lqZzcuFxQ">
+                            <img src="../image/btn-pre-youtube.png" class="img-center" />
+                        </a>
+                    </div>
+                    <div class="col-sm-1 hidden-xs">
+                        <a class="vcenter" href="https://instagram.com/amgguatemala/">
+                            <img src="../image/btn-pre-instagram.png" class="img-center" />
+                        </a>
+                    </div>
+                    <div class="col-sm-1 col-sm-offset-1">
+                        <a class="vcenter" href="en.amgg.org">English</a>
+                    </div>
+                    <div class="col-sm-1">
+                        <a class="vcenter" href="pdf/boletin.pdf">Boletin</a>
+                    </div>
+                    <div class="col-sm-1">
+                        <a class="vcenter" href="https://amgg.org/children/">Donar</a>
+                    </div>
+                    <div class="col-sm-2 hidden-xs">
+                        <a class="vcenter" href="https://www.facebook.com/Guates-Best-Store-1431567437168029/">Guates Best</a>
+                    </div>
+                    <div class="col-sm-1 pre-active">
+                        <a class="vcenter" href="involucrate.html#Apadrina">Apadrina</a>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <div class="content-wrapper row row-centered">
+            <div class="form col-md-6" id="children-to-sponsor">
+                <span class="header">Children You Are Sponsoring</span>
+                <hr>
+                <div class="col-md-4">
+                    <img class="center-block" src="../image/nino-2.png"/>
+                </div>
+                <span class="col-md-8 child-info"> Name </span>
```

Comment:
these are probably all going to need unique identifiers, should be ok for now. client side code will probably a lot easier to write if we have them though.